src/main/java/com/hazelcast/cloud/ClientWithSsl.java

```diff
@@ -68,299 +63,80 @@ public static void main(String[] args) throws Exception {
-        }
-        System.out.println("--------------------");
-    }
-
-    private static void createMappingForCities(SqlService sqlService) {
-        // See: https://docs.hazelcast.com/hazelcast/5.2-snapshot/sql/mapping-to-maps#compact-objects
+    private static void createMapping(SqlService sqlService) {
+        // See: https://docs.hazelcast.com/hazelcast/latest/sql/mapping-to-maps#compact-objects
         System.out.println("Creating mapping for cities...");
 
         String mappingSql = ""
-            + "CREATE OR REPLACE MAPPING city("
-            + " __key INT ,"
-            + " country VARCHAR ,"
-            + " city VARCHAR,"
-            + " population INT)"
-            + " TYPE IMap"
+            + "CREATE OR REPLACE MAPPING cities("
+            + "     __key INT,"
+            + "     country VARCHAR,"
+            + "     city VARCHAR,"
+            + "     population INT"
+            + ") TYPE IMap"
             + " OPTIONS ("
             + "     'keyFormat' = 'int',"
             + "     'valueFormat' = 'compact',"
             + "     'valueCompactTypeName' = 'city'"
             + " )";
 
         try (SqlResult ignored = sqlService.execute(mappingSql)) {
-            System.out.println("Mapping for cities has been created");
+            System.out.print("OK.");
         }
-        System.out.println("--------------------");
     }
 
-    private static void populateCities(HazelcastInstance client) {
-        // See: https://docs.hazelcast.com/hazelcast/5.2-snapshot/data-structures/creating-a-map
-        System.out.println("Populating 'city' map");
-
-        IMap<Integer, City> cities = client.getMap("city");
-        cities.put(1, new City("AU", "Canberra", 467_194));
-        cities.put(2, new City("CZ", "Prague", 1_318_085));
-        cities.put(3, new City("EN", "London", 9_540_576));
-        cities.put(4, new City("US", "Washington, DC", 7_887_965));
-
-        System.out.println("The 'city' map has been populated.");
-        System.out.println("--------------------");
-    }
+    private static void insertCities(HazelcastInstance client){
+        System.out.print("\nInserting cities into 'cities' map...");
+
+        String insertQuery = "INSERT INTO cities "
+                +"(__key, city, country, population) VALUES"
+                +"(1, 'London', 'United Kingdom', 9540576),"
+                +"(2, 'Manchester', 'United Kingdom', 2770434),"
+                +"(3, 'New York', 'United States', 19223191),"
+                +"(4, 'Los Angeles', 'United States', 3985520),"
+                +"(5, 'Istanbul', 'Türkiye', 15636243),"
+                +"(6, 'Ankara', 'Türkiye', 5309690),"
+                +"(7, 'Sao Paulo ', 'Brazil', 22429800)";
+
+        try{
+            SqlResult result = client.getSql().execute(insertQuery);
+            System.out.print("Inserted...");
+        }catch (Exception ex){
+            System.out.println(ex.getMessage());
+        }
```

Comment:
Why do we need to guard this with a separate try-catch?

Reply:
Also, I believe the log message can be more clear by finishing that sentence

Reply:
Since we insert, in multiple runs, client gets duplicated key exception. So, it prevents the application crashing, and continue to fetch cities.

Reply:
I see. Then, I believe it would be better to use SINK_INTO here, rather than showing an exception to the user on repeated runs. What do you think? https://docs.hazelcast.com/hazelcast/latest/sql/sink-into#difference-between-insert-into-and-sink-into

Reply:
I would say yes in the production but I'm not sure how it is correct to show different things in the first contact with user. The `INSERT` is so common that no body needs an explanation.

Reply:
But, seeing an error when you re-run the example is not a great experience IMO

Reply:
We can ignore the exception and don't print it. How does it sound?

Reply:
I am not sure, ignoring the exception doesn't look good to me(for example, what if it was really an exception?), but you are right about the `INSERT` being the widely known alternative.

I am not going to insist on this, but if I were to do this, I would use SINK INTO without exception handling, and add a small comment explaining its difference to INSERT